### PR TITLE
chore: Recommend `python -m asyncio` instead of Jupyter Notebook

### PR DESCRIPTION
**Pull request by kumaraditya303**

### CODE REVIEW

**Comment by mxschmitt** [comment]
I tested it out and it seems really helpful that this works with await out of the box, good find!

**Comment by kumaraditya303** [comment]
@pavelfeldman The failing tests are because of some network timeout and this PR only modifies docs

File: README.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ with sync_playwright() as p:
 #### Async API
 
 If you app is based on the modern asyncio loop and you are used to async/await constructs,
-Playwright exposes Async API for you. You should also use this API inside the Jupyter Notebook
-and other REPL frameworks that are already based on asyncio.
+Playwright exposes Async API for you. You should use this API inside a Python REPL supporting `asyncio` like with `python -m asyncio`
 
 ```py
+$ python -m asyncio
```
Comment (pavelfeldman) [inline README.md:72]: Looks like it does not look $ in python snippet

**Comment by kumaraditya303** [comment]
ping @pavelfeldman

**Comment by mxschmitt** [comment]
@kumaraditya303 I think when you cleanup the PR, so it contains only the asyncio changes, then it can get merged.

**Comment by kumaraditya303** [comment]
To speed up the process of merging PR the maintainers should add [CODEOWNERS](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) and may be configure auto merge feature of github @mxschmitt

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**